public/docs/ts/latest/guide/pipes.jade

```diff
@@ -39,6 +39,14 @@ block includes
   [pipe operator](./template-syntax.html#pipe) ( | ) to the [Date pipe](../api/common/index/DatePipe-class.html)
   function on the right. All pipes work this way.
 
+.l-sub-section
+  :marked
+    The `Date` and `Currency` pipes needs the **ECMAScript Internationalization API**.
+    Safari and other older browsers doesn't support it. We can add support with a polyfill.
```

Comment:
Should be "don't" not "doesn't".  But I'm not fixing either because I believe that @DeborahK has removed this section in an upcoming PR. No need to create a potential merge conflict.